flink-streaming-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer.java

```diff
@@ -632,6 +632,7 @@ public void run() {
 						//  ------------  commit current offsets ----------------
 
 						// create copy of current offsets
+						//noinspection unchecked
```

Comment:
Its only supressing IntelliJ warnings I think. I don't know if we have any policy regarding this. The annotation will also suppress compiler warnings.